src/NovelRT/CMakeLists.txt

```diff
@@ -193,6 +195,9 @@ elseif(APPLE)
   add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy
           $<TARGET_FILE:tbb>
           $<TARGET_FILE_DIR:Engine>/$<TARGET_FILE_NAME:tbb>)
+  add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy
+          $<TARGET_FILE:zlib>
+          $<TARGET_FILE_DIR:Engine>/$<TARGET_FILE_NAME:zlib>)
```

Comment:
is this duplicated?

Reply:
It was purposefully but I think I can condense it